src/main/java/seedu/edulog/ui/StudentCard.java

```diff
@@ -55,5 +64,119 @@ public StudentCard(Student student, int displayedIndex) {
         student.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        paidStatusIcon.setImage(student.getHasPaid() ? paidIcon : unpaidIcon);
```

Comment:
does breaking this up into if-else statements help with the incomptability?

Reply:
Unfortunately no :(